build.sbt

```diff
@@ -12,16 +13,15 @@ val akkaCluster = Seq(
   "com.typesafe.akka" %% "akka-cluster-tools" % AkkaVersion
 )
 
-(ThisBuild / playBuildRepoName) := "play-socket.io"
-(ThisBuild / sonatypeProfileName) := "com.lightbend"
+// Customise sbt-dynver's behaviour to make it work with tags which aren't v-prefixed
+(ThisBuild / dynverVTagPrefix) := false
 
 lazy val root = (project in file("."))
-  .enablePlugins(PlayReleaseBase, PlayLibrary)
   .settings(
-    organization := "com.lightbend.play",
+    organization := "com.typesafe.play",
```

Comment:
Because there is a beta release already (https://repo1.maven.org/maven2/com/lightbend/play/play-socket-io_2.12/) lets notify scala-steward: https://github.com/scala-steward-org/scala-steward/pull/2604

Reply:
👍